entry-asar/has-asar.js

```diff
@@ -0,0 +1,7 @@
+if (process.arch === 'arm64') {
+  process._archPath = require.resolve('../app-arm64.asar');
+} else {
+  process._archPath = require.resolve('../app-x64.asar');
+}
+
+require(process._archPath);
```

Comment:
`prettier` should have done this for us but can you add the trailing new line in here?